tests/test_emulators.py

```diff
@@ -4,7 +4,11 @@
 import pytest
 import xarray as xr
 from packaging.version import Version
-from raven_hydro import __raven_version__
+
+try:
+    from raven_hydro import __raven_version__
+except ImportError:
+    __raven_version__ = "0"
```

Comment:
How would you feel if we were to add a warning here?
```suggestion
try:
    from raven_hydro import __raven_version__
except ImportError:
    warnings.warn(UserWarning, "The `raven-hydro` library is not installed. Setting '__raven_version__' to '0'.")

    __raven_version__ = "0"
```